pkg/controller/stacks/install/installjob.go

```diff
@@ -250,8 +267,7 @@ func (jc *stackInstallJobCompleter) createJobOutputObject(ctx context.Context, o
 		"name", obj.GetName(),
 		"namespace", obj.GetNamespace(),
 		"apiVersion", obj.GetAPIVersion(),
-		"kind", obj.GetKind(),
-		"ownerRefs", obj.GetOwnerReferences())
```

Comment:
want to log the labels now, instead of the owner refs? this is debug level, so adding lots of info is good :)

Reply:
I've added this in 49a8ce58